src/js/jsx/sections/layers/LayerGroup.jsx

```diff
@@ -103,10 +103,7 @@ define(function (require, exports, module) {
                     );
 
                     var className = classnames({
-                        "layer-group": layerGroup,
-                        "layer-group__selected": layerGroup && layer.selected,
-                        "layer-group__collapsed": layerGroup && !layer.expanded,
-                        "layer-group__not-visible": layerGroup && !layer.visible
```

Comment:
I have to move these classes into `LayerFace` due to the `selected/expanded/visible` attributes is not always updated.